samples/snippets/csharp/VS_Snippets_CLR/conceptual.formatting.overview/cs/iformattable.cs

```diff
@@ -69,6 +69,7 @@ public class Example
 {
    public static void Main()
    {
+      System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("en-US");
```

Comment:
Since the example already imports the System.Globalization namespace, we can do this less verbosely with:
```
CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo("en-US");
```

Reply:
Thanks for pointing out the import, I oversaw it.
I did the PR for the VB snippets in parallel to this one: https://github.com/dotnet/docs/pull/3159

Reply:
Thanks, @paulroho. I noticed your PR after I reviewed this one. I'll quickly review it right now.